Command/SyncParametersCommand.php

```diff
@@ -69,11 +63,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 ($set_value === null ? 'has no value.' : var_export($set_value, true))
```

Comment:
Should be `$setValue` and can you fix other places where underscore is used in variable names?